src/harissa/core/model.py

```diff
@@ -205,24 +206,35 @@ def simulate(self,
             # Activate the stimulus
             initial_state[1, 0] = 1.0
         else:
-            if (not isinstance(initial_state, np.ndarray) 
+            if (not isinstance(initial_state, np.ndarray)
                 or initial_state.shape != state_shape):
                 raise TypeError(('initial_state must be a 2D np.ndarray '
                                 f'of shape {state_shape}.'))
             initial_state = initial_state.copy()
 
+        # Stimulus
+        if stimulus is None:
+            stimulus = initial_state[1, 0] * np.ones(time_points.shape)
+        else:
+            if (not isinstance(stimulus, np.ndarray)
+                or stimulus.shape != time_points.shape):
+                raise TypeError(('stimulus must be a 1D np.ndarray with '
+                                 'same size as time_points (or None).'))
+            stimulus = stimulus.copy()
+
```

Comment:
Should we also copy the first element of stimulus into initial_state[1, 0] to sync them?
```python
initial_state[1, 0] = stimulus[0]
```

Reply:
I added it